x-pack/platform/packages/private/security/ui_components/src/kibana_privilege_table/feature_table.tsx

```diff
@@ -353,7 +353,7 @@ export class FeatureTable extends Component<Props, State> {
             featureName: feature.name,
           },
         })}
-        style={{
+        css={{
```

Comment:
Replaced style with prefered css property. `minWidth` is required to properly render the feature access level options (all, read, none) in the feature table.